src/Molecules/Editable/Editable.jsx

```diff
@@ -46,26 +46,35 @@ const Editable = ({
   label = '',
   Type = Input,
   value = '',
+  labelSize = 'title',
+  descriptionSize = 'title',
+  descriptionFontFamily = '',

```

Comment:
Are we going to need to tweak those values ? Don't we want those to be static for general visual harmony purpose ?

Reply:
In her mockup, Tiff used a different typo that the one you used at first.
I don't know if this component was already used somewhere, so I decided to not modify its default typo, but to make it "editable" ;)